src/test/java/org/openrewrite/java/testing/junit5/CsvSourceToValueSourceTest.java

```diff
@@ -0,0 +1,457 @@
+
+              class TestClass {
+                  @ParameterizedTest
+                  @ValueSource(strings = "apple")
+                  void testWithSingleString(String fruit) {
+                      System.out.println(fruit);
+                  }
+              }
+              """
+          )
+        );
+    }
+
+    @Test
+    void replaceCsvSourceWithValueAttribute() {
+        rewriteRun(
+          //language=java
+          java(
+            """
+              import org.junit.jupiter.params.ParameterizedTest;
+              import org.junit.jupiter.params.provider.CsvSource;
+
+              class TestClass {
+                  @ParameterizedTest
+                  @CsvSource(value = {"test1", "test2", "test3"})
+                  void testWithValueAttribute(String value) {
+                      System.out.println(value);
+                  }
+              }
+              """,
+            """
+              import org.junit.jupiter.params.ParameterizedTest;
+              import org.junit.jupiter.params.provider.ValueSource;
+
+              class TestClass {
+                  @ParameterizedTest
+                  @ValueSource(strings = {"test1", "test2", "test3"})
+                  void testWithValueAttribute(String value) {
+                      System.out.println(value);
+                  }
+              }
+              """
+          )
+        );
+    }
+
+    @Test
+    void replaceCsvSourceWithChars() {
+        rewriteRun(
+          //language=java
+          java(
+            """
+              import org.junit.jupiter.params.ParameterizedTest;
+              import org.junit.jupiter.params.provider.CsvSource;
+
+              class TestClass {
+                  @ParameterizedTest
+                  @CsvSource({"a", "b", "c"})
+                  void testWithChars(char letter) {
+                      System.out.println(letter);
+                  }
+              }
+              """,
+            """
+              import org.junit.jupiter.params.ParameterizedTest;
+              import org.junit.jupiter.params.provider.ValueSource;
+
+              class TestClass {
+                  @ParameterizedTest
+                  @ValueSource(chars = {'a', 'b', 'c'})
+                  void testWithChars(char letter) {
+                      System.out.println(letter);
+                  }
+              }
+              """
+          )
+        );
+    }
+
+    @Test
+    void doNotReplaceWhenArgumentsAccessorIsUsed() {
+        rewriteRun(
+          //language=java
+          java(
+            """
+              import org.junit.jupiter.params.ParameterizedTest;
+              import org.junit.jupiter.params.aggregator.ArgumentsAccessor;
+              import org.junit.jupiter.params.provider.CsvSource;
+
+              class TestClass {
+                  @ParameterizedTest
+                  @CsvSource({"apple, 1", "banana, 2", "cherry, 3"})
```

Comment:
Shouldn't this be with one CSV line only? Related to https://github.com/openrewrite/rewrite-testing-frameworks/pull/797#issuecomment-3213746200:

```suggestion
                  @CsvSource("apple, 1")
```